lib/game.rb

```diff
@@ -1,36 +1,24 @@
-require 'pry'
-class Game
-  puts "In Game Class"
-  def set_player1(name, score = 0)
-    @player1 = name
-    # @score1 = []
-    # @score1 << score
-  end
+require_relative 'tennis'
 
-  def get_player1
-    @player1
-    @score1
-  end
+def capture_name
+  puts "Player 1 name: "
+  @player1 = gets.chomp
+
+  puts "Player 2 name: "
+  @player2 = gets.chomp
+end
+
+capture_name()
```

Comment:
Minor comment, but you don't need to put empty brackets around `capture_name`. By default it will look for a method with zero parameters.